src/replication.cc

```diff
@@ -499,6 +499,7 @@ ReplicationThread::CBState ReplicationThread::incrementBatchLoopCB(
         // Read bulk data (batch data)
         if (self->incr_bulk_len_+2 <= evbuffer_get_length(input)) {  // We got enough data
           bulk_data = reinterpret_cast<char *>(evbuffer_pullup(input, self->incr_bulk_len_ + 2));
+          self->incr_state_ = Incr_batch_size;
```

Comment:
maybe we should reset this state in the replication start function? or it may confuse why MUST reset the state before write batch